docs/website/pages/en/index.js

```diff
@@ -0,0 +1,214 @@
+/**
```

Comment:
We'd like this page to be as simple as possible. Like this one: https://linaria.now.sh/. Feel free to discard this comment, we first need to design it :D. But would be cool if you could remove all the unnecessary stuff and render only HomeSplash

Reply:
I've commented out the other sections and left in a link the example.md

It now has the website title, slogan, logo, and a button link to an example

Reply:
Would be even better if you removed this code, it's gonna be deleted anyway :)